content/news/XXXX-XX-XX-build-infrastructure-updates.md

```diff
@@ -0,0 +1,73 @@
+title: "Improving the Mixxx developer experience with maintainable build infrastructure"
+authors: Be.
+tags: 2.3, builds, infrastructure
+status: draft
+
+Since we [published Mixxx 2.3 beta in June 2020](/news/2020-06-07-mixxx-2-3-beta-released/), you may have been wondering what has taken so long to get the 2.3.0 release out. Well, we have been very busy the past few months building a more sustainable technical infrastructure for the Mixxx project. We have [moved this website to the Pelican static site generator](/news/2020-12-15-website-improvents/) and moved the [forums](https://mixxx.discourse.group/) to [Discourse](https://www.discourse.org/). Additionally, we had to do a ton of work for the macOS and Windows builds. "Building" software is the process of transforming the code that we write into 0s and 1s that tell your computer what to do.
```

Comment:
```suggestion
Since we [published Mixxx 2.3 beta in June 2020]({filename}/news/2020-06-07-mixxx-2-3-beta-released.md), you may have been wondering what has taken so long to get the 2.3.0 release out. Well, we have been very busy the past few months building a more sustainable technical infrastructure for the Mixxx project. We have [moved this website to the Pelican static site generator]({filename}/news/2020-12-15-website-improvents.md) and moved the [forums](https://mixxx.discourse.group/) to [Discourse](https://www.discourse.org/). Additionally, we had to do a ton of work for the macOS and Windows builds. "Building" software is the process of transforming the code that we write into 0s and 1s that tell your computer what to do.
```

... And everywhere else. That way pelican is able to check the validity of links during build.